### ISSUE TITLE
Get lib issues while trying to use the module

### ISSUE THREAD

**Issue by jinxac** [noderaider/redux-idle-monitor#6]
Getting following error while trying to use this package in my project:-

```
ERROR in ./~/redux-idle-monitor/lib/context.js
Module not found: Error: Cannot resolve module 'babel-runtime/core-js/json/stringify' in /Users/aragorn/taxo/node_modules/redux-idle-monitor/lib
 @ ./~/redux-idle-monitor/lib/context.js 7:17-64

ERROR in ./~/redux-idle-monitor/lib/context.js
Module not found: Error: Cannot resolve module 'babel-runtime/core-js/object/assign' in /Users/aragorn/taxo/node_modules/redux-idle-monitor/lib
 @ ./~/redux-idle-monitor/lib/context.js 11:14-60

ERROR in ./~/redux-idle-monitor/lib/reducer.js
Module not found: Error: Cannot resolve module 'babel-runtime/core-js/object/assign' in /Users/aragorn/taxo/node_modules/redux-idle-monitor/lib
 @ ./~/redux-idle-monitor/lib/reducer.js 8:14-60

ERROR in ./~/redux-idle-monitor/lib/middleware.js
Module not found: Error: Cannot resolve module 'babel-runtime/helpers/toConsumableArray' in /Users/aragorn/taxo/node_modules/redux-idle-monitor/lib
 @ ./~/redux-idle-monitor/lib/middleware.js 8:26-76

ERROR in ./~/redux-idle-monitor/lib/actions.js
Module not found: Error: Cannot resolve module 'babel-runtime/helpers/typeof' in /Users/aragorn/taxo/node_modules/redux-idle-monitor/lib
 @ ./~/redux-idle-monitor/lib/actions.js 8:15-54
```

Also looks like its quite heavy, my app size increase by ~0.3mb after adding this. Can you please help me on this?

**Comment by cchamberlain** [comment]
Thanks for reporting!

Can you type the following commands from your project root and paste the output?

`node --version`
`npm -v`
`npm ls redux-idle-monitor`
`npm ls babel-cli`
`npm ls babel-runtime`

**Comment by jinxac** [comment]
babel-runtime was not installed, shouldnt it be dependency in redux-idle-monitor. I tried installing babel-runtime , after which it worked fine. But still package is quite heavy :)

**Comment by cchamberlain** [comment]
Good catch, published v0.4.5 with updated dependencies. I've opened a new issue with regarding package size, I should be able to shave off some bloat in near future.
